src/controllers/user/display.js

```diff
@@ -24,7 +24,7 @@ import autoFocuser from 'autoFocuser';
             if (settingsInstance) {
                 settingsInstance.loadData();
             } else {
-                settingsInstance = new DisplaySettings({
+                settingsInstance = new DisplaySettings.default({
```

Comment:
```suggestion
                settingsInstance = new DisplaySettings({
```
I got `_displaySettings.default.default`. :man_shrugging: